packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -10,24 +10,25 @@ import 'package:flutter/material.dart';
 
 import '../../extensions/extension_screen.dart';
 import '../../extensions/extension_service.dart';
+import '../../service/editor/api_classes.dart';
+import '../../service/editor/editor_client.dart';
 import '../../shared/analytics/analytics.dart' as ga;
 import '../../shared/analytics/constants.dart' as gac;
 import '../../shared/constants.dart';
 import '../../shared/feature_flags.dart';
 import '../../shared/screen.dart';
-import '../api/vs_code_api.dart';
 
 class DebugSessions extends StatelessWidget {
   const DebugSessions({
-    required this.api,
+    required this.editor,
     required this.sessions,
-    required this.deviceMap,
+    required this.devices,
     super.key,
   });
 
-  final VsCodeApi api;
-  final List<VsCodeDebugSession> sessions;
-  final Map<String, VsCodeDevice> deviceMap;
+  final EditorClient editor;
+  final Map<String, EditorDebugSession> sessions;
+  final Map<String, EditorDevice> devices;
```

Comment:
can you add dart doc for these? What is the string representing in the map?

Reply:
Done in 8ec9692b9 (the keys are the IDs).